internal/website/docs/next/reference/hooks/usePreview.mdx

```diff
@@ -4,6 +4,12 @@ title: usePreview Hook
 description: The usePreview hook provides a way to get post preview data from WordPress
 ---
 
+:::caution Deprecation Notice
+
+`usePreview` has been deprecated in favor of [`usePreviewNode`](/docs/next/reference/hooks/usePreviewNode)
```

Comment:
Just wondering if its possible to log a deprecated usage warning whenever a client tries to use the `usePreview`?

Reply:
This is a great idea. I'll add this.

Reply:
On second thought, we may not be able to do this nicely. We could log a message in the `usePreview` hook, but it would log the message on every re-render and start to pollute the console.